extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -2958,50 +2931,9 @@ test('isHypervEnabled should return true if hyperv is enabled', async () => {
   expect(wslHypervEnabled).toBeTruthy();
 });
 
-test('isWSLEnabled should return false if it is not windows', async () => {
-  vi.mocked(extensionApi.env).isWindows = false;
-  const wslEnabled = await extension.isWSLEnabled();
-  expect(wslEnabled).toBeFalsy();
-});
-
-test('isWSLEnabled should return false if wsl is not enabled', async () => {
-  vi.mocked(extensionApi.env).isWindows = true;
-  vi.spyOn(extensionApi.process, 'exec').mockResolvedValue({
-    stdout: 'unknown message: 1.2.5.0',
-    stderr: '',
-    command: 'command',
-  });
-  const wslEnabled = await extension.isWSLEnabled();
-  expect(wslEnabled).toBeFalsy();
-});
```

Comment:
This scenario is also covered by `wsl-version-check.spec.ts`

https://github.com/podman-desktop/podman-desktop/blob/49ea010784b60d857cc240079866be8f0cf7685d/extensions/podman/packages/extension/src/checks/windows/wsl-version-check.spec.ts#L60-L73